src/components/ArticleVotes.jsx

```diff
@@ -1,11 +1,43 @@
 import { IconButton, Stack, SvgIcon } from "@mui/material";
+import { updateArticle } from "../utils";
+import { useState } from "react";
+import Button from "@mui/material/Button";
+import Snackbar from "@mui/material/Snackbar";
+import MuiAlert from "@mui/material/Alert";
+import * as React from "react";
 
-export default function ArticleVotes({ votes, setVotes }) {
+const Alert = React.forwardRef(function Alert(props, ref) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
+});
+
+export default function ArticleVotes({ article_id, votes, setVotes }) {
+  const [errorOpen, setErrorOpen] = useState(false);
+  const handleErrorClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+
+    setErrorOpen(false);
+  };
+  const handleUpVote = () => {
+    setVotes(1);
+    updateArticle(article_id, { incVotes: 1 }).catch(() => {
```

Comment:
Nice bit of optimistic rendering here

Reply:
If I so wished, as a user I can smash the votes up and down as much as I like...
YOu might want to put some functionality in to prevent this.